### DISCUSSION TITLE
It would be really helpful to know what order the fans are in!

### DISCUSSION THREAD

**Discussion by tylerjburns** [ts-korhonen/LibreHardwareMonitorAfterburnerPlugin#5] [Ideas]
It would be very helpful to include the order of the fans, or label them somehow. I'm not sure if the software has access to that information, but I would think the mobo would provide that info somehow. I could also try to contribute if you're looking for some support on this!

**Comment by ts-korhonen** [comment]
The names come from the [LibreHardwareMonitorLib](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor). When looking at [this file in the source code](https://github.com/LibreHardwareMonitor/LibreHardwareMonitor/blob/master/LibreHardwareMonitorLib/Hardware/Motherboard/SuperIOHardware.cs), it seems like some have actual names for the fans for certain motherboards and unknown boards get just numbers.

So the best way to fix that is to contribute to that project.
